bids-validator/validators/bids/groupFileTypes.js

```diff
@@ -52,6 +53,9 @@ const sortFiles = (fileList, options, files) => {
       files.misc.push(file)
     } else if (ofType(filename, 'tsv.gz')) {
       files.contRecord.push(file)
+    } else if (ofType(filename, 'ome.tif')) {
```

Comment:
Sorry, I think I forgot to mention it before but OMETIFF files have 2 valid extension in Microscopy-BIDS:
`.ome.tif` and  `.ome.btf`

Reply:
Done